packages/components/src/table/index.js

```diff
@@ -268,12 +276,13 @@ class TableCard extends Component {
 							{ labels.compareButton || __( 'Compare', 'wc-admin' ) }
 						</CompareButton>
 					),
-					compareBy && (
+					( compareBy || searchBy ) && (
 						<Search
 							key="search"
 							placeholder={ labels.placeholder || __( 'Search by item name', 'wc-admin' ) }
```

Comment:
Should we pass in props above to change this to `Search by customer name`?

Reply:
Absolutely! Fixed: 416bba29ea3cfa633caefac55b41ca351157caa0